### PR TITLE
Fix some typo in 2015-07

### PR DESCRIPTION
**Pull request by azu**
This pull request fixes some typo and markdown syntax.

----

@rwaldron Thank you for your work! 
 I'm always looking forward to seeing meeting note :memo: :)

### CODE REVIEW

File: es7/2015-07/july-29.md [hunk 1]
```diff
@@ -363,7 +363,7 @@ With Sam's proposa, this would return `[undefined, Global]`, despite expecting `
   "use strict";
   return [this, foo];
 })();
-
+```
 
 DH: In ES5, there was a fundamental confusion about "use strict" in functions. This was excacerbated by es6. In the fullness of time, TC39 is saying, the directive prologue should be used:
```
Comment (ljharb) [inline es7/2015-07/july-29.md]: s/excacerbated/exacerbated
s/es6/ES6

while you're at it :-)

Comment (azu) [inline es7/2015-07/july-29.md]: fixed in 8a85fd5

**Comment by azu** [comment]
ping @rwaldron

**Comment by rwaldron** [comment]
Thanks!!

**Pull request merged**